ReceiverThread.cpp

```diff
@@ -6,13 +6,13 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  */
-#include <wdt/ReceiverThread.h>
 #include <folly/Bits.h>
 #include <folly/Checksum.h>
 #include <folly/Conv.h>
 #include <folly/Memory.h>
 #include <folly/ScopeGuard.h>
 #include <folly/String.h>
+#include <wdt/ReceiverThread.h>
```

Comment:
Revert this one too.

"The associated header file of .cpp files should be included before any other includes. (This helps catch missing header file dependencies in the .h)"

This is the lint error I'm seeing.